webapp/src/components/Vault/Modal/YourPositionModal.tsx

```diff
@@ -123,14 +123,14 @@ const YourPositionModal: React.FC = () => {
             {/* Logo */}
             <BaseModalContentColumn>
               <AssetCircleContainer size={96} color={color}>
-                <Logo width={48} height={48} />
+                <Logo width={"100%"} height={"100%"} />
```

Comment:
@fqlx @kenchangh @cjinghong , we should just resize a `Logo` within an `AssetCircleContainer` to fit rather than manually tweaking the size. If there are any manual tweaks that need to be done, we should just pass in a `padding` value rather than adjusting the `width` and `height`.